pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -67,6 +68,7 @@ let
         ([ ]
           ++ lib.optional (cfg.enableBrowserpass or false) (lib.getBin browserpass)
           ++ lib.optional (cfg.enableBukubrow or false) bukubrow
+          ++ lib.optional (cfg.enableTridactylNative or false) tridactyl-native
```

Comment:
One more thing though: I think the `Native` part can be dropped, it's also not used for all other packages. Same for the package attribute `tridactyl-native -> tridactyl`, package path and name too.

Reply:
I disagree: if we ever get firefox plugin support in nixpkgs, it would be confusing to already have a package called `tridactyl`. It should be clear for the user what is being installed and what is not (they will have to install the actual plugin through other means).

Do you insist on the name change?

Reply:
Hmm, but then it's still the tridactyl package from the same repo, and we'd probably just make it have multiple outputs, one for the native part, one for the extension part.

Reply:
But shouldn't we still separate it then? There is good reason firefox doesn't install native messengers itself, it basically sidesteps the whole permission mechanism.

Reply:
Usage with separate derivations for native and extension:

```nix
{ pkgs }: {
  nativePart = pkgs.tridactyl-native;
  extensionPart = pkgs.tridactyl-extension;
}
```

Usage with a single derivation using multiple outputs:
```nix
{ pkgs }: {
  nativePart = pkgs.tridactyl.native; # Or pkgs.tridactyl because it's the default output
  extensionPart = pkgs.tridactyl.extension;
}
```

I don't see how the latter could be a problem, they really only differ by `s/-/.`.

Reply:
Yes, but shouldn't `extensionPart` be the default? When I install `tridactyl`, I expect to be installing the plugin. I'd be surprised if it (only) installed the native messenger by default.

Reply:
If we ever do get support for packaging extensions, I'd guess we'd put it in `browserExtensions.tridactyl` or so anyways, and you could install the native one via `pkgs.tridactyl`. Anyways, if you really think we should leave the -native part in the name then so be it.

Reply:
I still do think that the `-native` postfix reduces confusion and I don't see any drawbacks to using it, so I'd rather keep it.

Thanks a lot for the review :)